CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_ad_reporting v1.13.0
+[PR #137](https://github.com/fivetran/dbt_ad_reporting/pull/137) includes the following updates:
+
+## Breaking Changes: Version Updates
+The following dependencies have been updated following upstream breaking changes. See the below release notes for more information.
+  - `dbt_apple_search_ads` ([v0.5.0](https://github.com/fivetran/dbt_apple_search_ads/releases/tag/v0.5.0))
+  - `dbt_microsoft_ads` ([v0.10.0](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.10.0))
+The `conversions` field for `apple_search_ads` now sources from `tap_installs`.
```

Comment:
The way this renders makes it look like this is part of the microsoft ads update
![image](https://github.com/user-attachments/assets/7d7086ec-3975-47b0-a002-b91e1a1a91ce)

Perhaps make it its own bullet or a sub-bullet of the apple search ads note? Or could go in the "Under the Hood" section

Reply:
thanks for catching! moved it to under the hood